pymammotion/http/http.py

```diff
@@ -71,7 +74,30 @@ async def login(cls, session: ClientSession, username: str, password: str) -> Re
                 response = Response.from_dict(data)
                 # TODO catch errors from mismatch user / password elsewhere
                 # Assuming the data format matches the expected structure
+                cls._session = session
                 return response
+    
+    @classmethod
+    async def get_stream_subscription(cls, iot_id: str) -> Response[StreamSubscriptionResponse]:
+        """Get agora.io data for view camera stream"""    
+        async with ClientSession('https://domestic.mammotion.com') as session:
```

Comment:
```suggestion
        async with ClientSession(DOMESTIC_MAMMOTION_URL) as session:
```

Turn this into a const alongside MAMMOTION_DOMAIN
'https://domestic.mammotion.com'

should probably have a mammotion domain that can take a subdomain since there are different subdomains